package.json

```diff
@@ -48,5 +48,8 @@
     "cross-env": "^7.0.3",
     "patch-package": "^6.4.7",
     "playwright": "1.40.0"
+  },
+  "dependencies": {
+    "comlink": "^4.4.1"
```

Comment:
i think this can go into the nimble-components workspace

Reply:
indeed